### DISCUSSION TITLE
Java exit code = 14

### DISCUSSION THREAD

**Discussion by aurorapar** [dbeaver/dbeaver#16161] [Q&A]
#### System information: 
- Operating system (distribution) and version // Microsoft Windows [Version 10.0.19043.1466]
- DBeaver version // Latest Release
- Java version // Java(TM) SE Runtime Environment (build 1.8.0_321-b07)
- Additional extensions // None

#### Describe the problem you're observing:
I installed DBeaver and decided to *not* check the Java install. When attempting to run DBeaver, it notified me that my Java version was out of date. I obtained a new Java version, and it prompted me to uninstall my old version. I allowed it to uninstall my previous version. Now every time I boot DBeaver I receive this error message.
![image](https://user-images.githubusercontent.com/2294961/152886447-41426dcf-366a-4b5f-8094-22b8635b0707.png)

#### Include any warning/errors/backtraces from the logs
I do not appear to have any log files. There is no Java error dump within `%LOCALAPPDATA%/DBeaver` , and the only files within `%LOCALAPPDATA%/DBeaver/configuration` are `config.ini`, `platforms.xml`, and `bundles.info`.

**Comment by aurorapar** [comment]
After installing JDK 11, DBeaver runs fine. I was under the assumption that only the runtime was required. Is this accurate?

**Comment by flambchap** [comment]
Hi @xxauroraxx. DBeaver requires JRE 11+. We include our custom JRE in most of our packages.
